### ISSUE TITLE
suddenly .d.dia raw files not loadable any more 

### ISSUE THREAD

**Issue by JM-Bader** [vdemichev/DiaNN#416]
I ran two searches in parallel, one with .d TimsTOF Pro raw files one with .dia version of the raw files - to compare the speed gain. 

The search with the .d raw files ran through. 
The search with .dia raw files started normally and at some point all following .dia files could not be loaded, were skipped with 0 peptides/proteins identified.  Subsequently, there were not enough peptides for normalisation. 
It appears unlikely that a network issue was responsible for that, because the other search with .d raw files did not have any issues. 
[DA01.14_PL01_DA-SER1.15_dia files_.log.txt](https://github.com/vdemichev/DiaNN/files/8965976/DA01.14_PL01_DA-SER1.15_dia.files_.log.txt)

![image](https://user-images.githubusercontent.com/98888664/175267341-97ab076a-e9d2-4039-9d59-ddb6ce60e734.png)

**Comment by vdemichev** [comment]
Yes, if files are on a network drive, if network is unstable, errors can happen.

**Comment by JM-Bader** [comment]
Q3: In #413 you mentioned that you would not recommend using .d.dia instead of .d - as .d is relatively well compressed anyway. Is there a disadvantage of using .d.dia (information loss?)?

Second, is there still a search speed gain when using .d.dia compared to normal .d? (Or is it marginal?)

**Comment by vdemichev** [comment]
The only disadvantage of .d.dia is space on disk. Yes, there's still a speed gain.
